.travis.yml

```diff
@@ -9,6 +12,8 @@ before_install:
     - sudo make install
     - sudo ldconfig /usr/local/lib
 
+before_script: "echo \"db.test.insert({ foo: 42 }); db.getSiblingDB('sample').movies.insert({ title: 'Jaws' });\" | mongo"
```

Comment:
I haven't adjusted this for your outstanding PR yet. The code in 1.0 right now assumes that there's a `{ foo: 42 }` document in 'test.test'